guessinggame.py

```diff
@@ -0,0 +1,25 @@
+import random
+
+number = random.randint(0, 10)
+guess = int(input("Enter an integer from 0 to 10: "))
+numofguesses = 0
+
+while number != guess:
```

Comment:
Wouldn't it be easier to use a for loop that runs 8 times? And finishes? Since the player can no longer enter guesses after trying 8 times?

Using a while loop means allowing the loop to run infinitely.

Reply:
yeah that makes more sense.